Gruntfile.js

```diff
@@ -65,6 +66,22 @@ module.exports = function (grunt) {
 		grunt.file.write('test/tmp/3');
 	});
 
+	grunt.registerTask('test4', function () {
+		console.log('test4');
+		grunt.file.write('test/tmp/4');
+	});
+
+	grunt.registerTask('test5', function () {
```

Comment:
The problem is I didn't want this task to be asynchronous. It hack simply blocks any other tasks for 1s so I can be sure that these are made one after another.